packages/apps/crm/src/screens/event/EventPlanningScreen.js

```diff
@@ -0,0 +1,259 @@
+import React, {useCallback, useMemo, useState, useEffect} from 'react';
+import {StyleSheet, TouchableOpacity, Dimensions, View} from 'react-native';
+import {
+  Card,
+  LabelText,
+  Screen,
+  Text,
+  useThemeColor,
+  HeaderContainer,
+  ChipSelect,
+  AutoCompleteSearch,
+  ToggleSwitch,
+  getCommonStyles,
+} from '@axelor/aos-mobile-ui';
+import {
+  PlanningView,
+  useDispatch,
+  useSelector,
+  useTranslator,
+  filterChip,
+} from '@axelor/aos-mobile-core';
+import {fetchPlannedEvent} from '../../features/eventSlice';
+import EventType from '../../types/event-type';
+
+function EventPlanningScreen({navigation}) {
+  const dispatch = useDispatch();
+  const Colors = useThemeColor();
+  const I18n = useTranslator();
+  const {eventList, loading} = useSelector(state => state.event);
+  const {userId} = useSelector(state => state.auth);
+  const [filteredList, setFilteredList] = useState(eventList);
+  const [assigned, setAssigned] = useState(false);
+  const [selectedStatus, setSelectedStatus] = useState([]);
+  const [plannedEvent, setPlannedEvent] = useState(null);
+  const commonStyles = useMemo(() => getCommonStyles(Colors), [Colors]);
+  const [dateSave, setDateSave] = useState(null);
+
+  useEffect(() => {
+    setFilteredList(filterOnUserAssigned(filterOnStatus(eventList)));
+  }, [
+    filterOnUserAssigned,
+    filterOnStatus,
+    eventList,
+    selectedStatus,
+    assigned,
+  ]);
+
+  const listItem = useMemo(() => {
+    return EventType.getCalendarListItems(filteredList, Colors);
+  }, [Colors, filteredList]);
+
+  const fetchEventFilter = useCallback(
+    searchValue => {
+      dispatch(fetchPlannedEvent({date: dateSave, searchValue: searchValue}));
+    },
+    [dispatch, dateSave],
+  );
+
+  const filterOnStatus = useCallback(
+    list => {
+      return filterChip(list, selectedStatus, 'typeSelect');
+    },
+    [selectedStatus],
+  );
+
+  const styles = useMemo(() => {
+    return getStyles(Colors);
+  }, [Colors]);
+
+  const fetchItemsByMonth = useCallback(
+    date => {
+      dateSave === null && setDateSave(date);
+      dispatch(fetchPlannedEvent({date: date}));
+    },
+    [dispatch, dateSave],
+  );
```

Comment:
Client filter disappear on refresh. fetchEventFilter should set the searchValue in a filter variable and this value should be re-used here to keep filter.